tasks.rb

```diff
@@ -141,6 +141,9 @@ def create_hpxmls
     'base-enclosure-windows-inoperable.xml' => 'base.xml',
     'base-enclosure-windows-interior-shading.xml' => 'base.xml',
     'base-enclosure-windows-none.xml' => 'base.xml',
+    'base-enclosure-wall-unrated-heated-space.xml' => 'base.xml',
+    'base-enclosure-two-wall-multifamily-buffer.xml' => 'base.xml',
+    'base-enclosure-wall-non-freezing-space.xml' => 'base.xml',
```

Comment:
Let's just put all the new spaces into a single test file that includes all the possible adjacency scenarios (once you handle foundation walls, rim joists, and frame floors), so we don't create a large proliferation of new test files.

Reply:
Could be called 'base-enclosure-attached-multifamily.xml'?